migrations/versions/10c965144f38_initial_schema.py

```diff
@@ -0,0 +1,26 @@
+"""initial schema
+
```

Comment:
I created this initial migration with the alembic autogenerate command to create the initial alembic models of each table to be compared against. Upgrade and downgrade both have passes which ensures that the autogenerated models look like the database